src/hooks/useTokenBalance.ts

```diff
@@ -61,4 +63,24 @@ export const useBurnedBalance = (tokenAddress: string) => {
   return balance
 }
 
+export const useGetBnbBalance = () => {
+  const [balance, setBalance] = useState(new BigNumber(0))
+  const { account } = useWeb3React()
+  const { lastUpdated, setLastUpdated } = useLastUpdated()
+
+  useEffect(() => {
+    const fetchBalance = async () => {
+      const web3 = getWeb3NoAccount()
+      const walletBalance = await web3.eth.getBalance(account)
+      setBalance(new BigNumber(walletBalance))
```

Comment:
Don't you need `web3.utils.fromWei(walletBalance, "ether")` here? Or it's supposed to be in Wei?

Reply:
I wasn't aware of that utility but I think most of our balances return as Wei.